SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -68,6 +79,8 @@ cd ..
 pwd
 # compile containers/image manpages
 cd image-%{imagever}
+# NOTE: Patch3 has to be applied as -p6
+patch -p6 < %{PATCH3}
```

Comment:
Please move all patching into `%prep`. That allows developers to work with all sources already patched after running `rpmbuild -bp`. That's very useful when we're applying new patches to the existing ones.

Another benefit of moving to `%prep`: you can use the `%patch` macro.

Reply:
That is a very good catch. Thanks, yes - it makes sense :)
Done!